src/SIL.XForge.Scripture/ClientApp/src/app/checking/checking/checking-audio-player-new/checking-audio-player-new.component.ts

```diff
@@ -0,0 +1,42 @@
+import { AfterViewInit, Component, Input, OnDestroy, ViewChild } from '@angular/core';
+import { AudioPlayerComponent } from 'src/app/shared/audio/audio-player.component';
+import { I18nService } from 'xforge-common/i18n.service';
+import { SubscriptionDisposable } from 'xforge-common/subscription-disposable';
+
+@Component({
+  selector: 'app-checking-audio-player-new',
+  templateUrl: './checking-audio-player-new.component.html',
+  styleUrls: ['./checking-audio-player-new.component.scss']
+})
+export class CheckingAudioPlayerNewComponent extends SubscriptionDisposable implements OnDestroy, AfterViewInit {
+  private _isAudioAvailable = false;
+  @ViewChild(AudioPlayerComponent) audioPlayer?: AudioPlayerComponent;
+  @Input() source: string = '';
+
+  constructor(readonly i18n: I18nService) {
+    super();
+  }
+
+  ngAfterViewInit(): void {
+    this.subscribe(this.audioPlayer!.isAudioAvailableChanged, () => {
+      setTimeout(() => (this._isAudioAvailable = this.audioPlayer!.enabled));
```

Comment:
Necessary to update asynchronously in order to avoid changing during a Change Detection cycle.